File: views/mokrij-fasad.go
package views

import (
	"html/template"
	"net/http"

	mobiledetect "github.com/Shaked/gomobiledetect"
	"github.com/meehalkoff/loggi"
	"gitlab.com/mvalkov/katzman/lib"
)

func MokrijFasadHandler(w http.ResponseWriter, r *http.Request) {

	tmpl := template.New("mokrij-fasad")

	detect := mobiledetect.NewMobileDetect(r, nil)
	if detect.IsMobile() {
		tmpl.ParseFiles(
			"mobile/templates/mokrij-fasad.html",
			"mobile/templates/header.html",
			"mobile/templates/footer.html",
			"mobile/templates/top-panel.html",
			"mobile/templates/calc.html",
			"mobile/templates/call.html",
			"mobile/templates/about.html",
			"mobile/templates/firstPromoBlock.html",
			"mobile/templates/creditPromoBlock.html",
		)
	} else {
		tmpl.ParseFiles(
			"templates/header.html",
			"templates/footer.html",
			"templates/firstPromoBlock.html",
			"templates/lastPromoBlock.html",
			"templates/creditPromoBlock.html",
			"templates/top-panel/top-panel.html",
			"templates/top-panel/about.html",
			"templates/top-panel/calc.html",
			"templates/top-panel/call.html",
			"templates/top-panel/main-menu.html",
			"templates/control-panel.html",
			"templates/mokrij-fasad/index.html",
			"templates/mokrij-fasad/vid-pokritij.html",
			"templates/mokrij-fasad/palitra.html",
			"templates/mokrij-fasad/plus-minus.html",
			"templates/mokrij-fasad/schema.html",
			"templates/mokrij-fasad/otkos.html",
			"templates/mokrij-fasad/portfolio.html",
			// "templates/shared/video.html",
			"templates/mokrij-fasad/video.html",
			"templates/mokrij-fasad/promo.html",
		)
	}

	data := lib.FirstPromoBlock{
		MaterialName: `Утепление фасадов, короед, шуба, мраморная крошка.`,
		ID:           "mokrijPrice",
		PictureURL:   "/assets/img/promo/koroed.jpg",
		UnitName:     "кв.м.",
		YoutubeID:    "n_nHmqCOLVw",
	}

	if err := tmpl.Execute(w, data); err != nil {
		loggi.Fatal(err)
	}
}

File: assets/js/mokrij-fasad.js
var nextArrow = '<div class="control-next-arrow">▶▶</div>';
var prevArrow = '<div class="control-prev-arrow">◀◀</div>';
var maxSlidePosition = 8;

var control = {
    0: {
        id: "first-promo",
        title: "Мокрый фасад: предложение",
        prev: "",
        next: "Виды покрытий" + nextArrow,
    },
    1: {
        id: "vid",
        title: "Мокрый фасад: виды покрытий",
        prev: prevArrow + "Предложение",
        next: "Палитра цветов" + nextArrow,
    },
    2: {
        id: "palitra",
        title: "Мокрый фасад: палитра цветов",
        prev: prevArrow + "Виды покрытий",
        next: "Плюсы и минусы" + nextArrow,
    },
    3: {
        id: "plus-minus",
        title: "Мокрый фасад: плюсы и минусы",
        prev: prevArrow + "Палитра цветов",
        next: "Технология монтажа" + nextArrow,
    },
    4: {
        id: "schema",
        title: "Мокрый фасад: технология монтажа",
        prev: prevArrow + "Плюсы и минусы",
        next: "Устройство откосов" + nextArrow,
    },
    5: {
        id: "otkos",
        title: "Мокрый фасад: устройство откосов",
        prev: prevArrow + "Технология монтажа",
        next: "Портфолио" + nextArrow,
    },
    6: {
        id: "portfolio",
        title: "Мокрый фасад: портфолио",
        prev: prevArrow + "Устройство откосов",
        next: "Видео обзор" + nextArrow,
    },
    7: {
        id: "video",
        title: "Мокрый фасад: видео обзор",
        prev: prevArrow + "Портфолио",
        next: "Предложение" + nextArrow,
    },
    8: {
        id: "last-promo",
        title: "Мокрый фасад: звоните!",
        prev: prevArrow + "Видео",
        next: "",
    },

}

var maxPortfolioPosition = 7;

var portfolio = {
    0: 'url(/assets/img/mokrij-fasad/portfolio/1.jpg)',
    1: 'url(/assets/img/mokrij-fasad/portfolio/2.jpg)',
    2: 'url(/assets/img/mokrij-fasad/portfolio/3.jpg)',
    3: 'url(/assets/img/mokrij-fasad/portfolio/4.jpg)',
    4: 'url(/assets/img/mokrij-fasad/portfolio/5.jpg)',
    5: 'url(/assets/img/mokrij-fasad/portfolio/6.jpg)',
    6: 'url(/assets/img/mokrij-fasad/portfolio/7.jpg)',
    7: 'url(/assets/img/mokrij-fasad/portfolio/8.jpg)',
}